Nodejs/Product/ProjectWizard/ImportWizard.cs

```diff
@@ -45,6 +46,8 @@ public sealed class NewProjectFromExistingWizard : IWizard {
                     dte = new ServiceProvider(provider).GetService(typeof(DTE)) as DTE;
                 }
             }
+
+            bool addingNewProject = false;
```

Comment:
No I used it at line 79. The one gets reset is the property, not the local variable.